app/controllers/user/trips_controller.rb

```diff
@@ -2,4 +2,30 @@ class User::TripsController < User::BaseController
   def index; end
 
   def new; end
+
+  def create
+    begin
+      trip = current_user.trips.create!(trip_params)
+      redirect_to edit_user_trip_path(trip)
+    rescue ActiveRecord::RecordInvalid
+      flash[:error] = errors.messages
+      render new
+    end
+  end
+
+  def edit; end
+
+  private
+
+  def trip_params
+    params.permit(:location, :start_date, :end_date)
+  end
+
+  def cuisine_params
+    params.permit(:indian, :mexican, :pizza, :italian, :american)
+  end
```

Comment:
These can been changed when we incorporate the micro service I believe right?

Reply:
Yes. The idea was to pass those onto our api so it can return relevant trip options.